index.bs

```diff
@@ -867,12 +867,16 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 										1. Resolve |p| with |v|.
 
+										1. Return |p|.
```

Comment:
Let's just make one change at a time focused on describing optional sanitization.  Could you remove these new returns and other editorial changes?